### ISSUE TITLE
Modify the Guest List CSV

### ISSUE THREAD

**Issue by yarunluon** [fnf-org/TicketBooth#144]
**Is your feature request related to a problem? Please describe.**
Gate needs a list of attendees that is formatted to expeditiously check-in attendees. They will be printing out the Guest List so need a quick way to scan for names. 

**Describe the solution you'd like**
Modify the Guest List CSV to use these columns per @mrwilldehaan 's [comment](https://github.com/fnf-org/TicketBooth/issues/144#issuecomment-2105344078)

* Name first last: _alphabetical ascending (a-z) order, concatenated_
* role
* early access (EA)
* email: _used as a backup to verify attendance if name can't be found. People use DJ names or Playa/FnF names. Can also be used as secondary verification._

FYI: Gate will add these columns which will help Gates and Tickets post-event for auditing tasks
* arrival time
* tent/car/rv + payment status

The Guest List CSV can be found under the "Guest List" tab in the Admin view
![image](https://github.com/fnf-org/TicketBooth/assets/1328337/71cc0a80-769d-4a71-9d4e-63e016629975)

**Describe alternatives you've considered**
Taken from Slack thread:
1. Tickets team will write some spreadsheet equations to generate a “Gate List” using the “Ticket Requests CSV”. 
2. Gate does the work instead of Tickets and reformats the “Ticket Requests CSV” or the “Guest List CSV” to their needs.
3. We do nothing and Gate needs to look up the attendee by Sponsor -> Guest name using the current “Guest List CSV”.

**Additional context**
The current format of the Guest List is
* Name (aka Sponsor or Ticket Requester)
* _n_-Guest names

### Format of Guest Names
![image](https://github.com/fnf-org/TicketBooth/assets/1328337/8c2d3f03-ff81-4cc0-b6b8-0253bd575c0d)

The guest list can be up to 4 pages long when printed. The majority of attendees will be in the "Guest" columns. Scanning a 4 page document to find guest names is time consuming. This will generate long queues of people in their cars.

**Comment by mrwilldehaan** [comment]
Gotta rule out #3 as not useful since guests and hosts show up in different vehicles

Gate list columns look like:
Name first last, role, early access (EA), arrival time, tent/car/rv + payment status, email

Gate can do data-entry to populate the database after the event if desired. Good job for a ravey intern

**Comment by yarunluon** [comment]
Thanks! Updated Issue description based on @mrwilldehaan 's comment.

**Comment by yarunluon** [comment]
In the version of TicketBooth that was > `v1.0.1`, the names of the guest list were required to enter the payment flow. This was enforced by disabling the link and displaying a message saying all guest names had to be filled out.

**Comment by mrwilldehaan** [comment]
Pretty hard to figure out what's going on here, but let me try again.

We need every attendees name on a separate row

**Comment by kigster** [comment]
I am super confused.

After the requirement to fill out the guest names was nuked, and now at no point in the ticket request flow is the ticket buyer required to fill out this info, what is the value of even including the guests when only a small portion of ticket holders would go back and fill that out?

**Comment by yarunluon** [comment]
> After the requirement to fill out the guest names was nuked, and now at no point in the ticket request flow is the ticket buyer required to fill out this info, what is the value of even including the guests when only a small portion of ticket holders would go back and fill that out?

I am now seeing this comment and would have responded earlier. See this [comment](https://github.com/fnf-org/TicketBooth/issues/144#issuecomment-2107985791). The old version of TicketBooth enforced Guest Names before entering the payment flow. When old Ticketbooth was migrated to new Ticketbooth, it was assumed this validation would remain the same. My fault. I should have explicitly called out that Guest Names are enforced before entering the payment flow.

**Comment by yarunluon** [comment]
> Name first last, role, early access (EA), arrival time, tent/car/rv + payment status, email

@mrwilldehaan , when you mention "tent/car/rv + payment status", are you referring to the payment status for Car Camping or for Tickets? I was assuming the former.

**Comment by mrwilldehaan** [comment]
The former. Payment status for cr and RV camping. 🌟 THANKS 🙏

**Comment by mrwilldehaan** [comment]
Ok if we're missing a guest name then the gate row could be like this:

Will DeHaan, coordinator, rv paid, ...
Will DeHaan Guest 1, sparkle pony, tent,...
Will DeHaan Guest 2, ...
